packages/schematics/angular/ng-new/schema.json

```diff
@@ -144,6 +144,16 @@
       "x-prompt": "Do you want to create a 'zoneless' application without zone.js (Developer Preview)?",
       "type": "boolean",
       "default": false
+    },
+    "aiConfig": {
+      "type": "array",
+      "default": [],
+      "uniqueItems": true,
+      "description": "Create an AI configuration file for the project. This file is used to improve the outputs of AI tools by following the best practices.",
+      "items": {
+        "type": "string",
+        "enum": ["gemini", "copilot", "claude", "cursor", "windsurf"]
+      }
```

Comment:
Question: Do we need `x-prompt` here too to show up as a prompt for just `ng new`?